pkg/frontend/frontend.go

```diff
@@ -76,37 +85,42 @@ type Runnable interface {
 	Run(context.Context, <-chan struct{}, chan<- struct{})
 }
 
-// NewFrontend returns a new runnable frontend
-func NewFrontend(ctx context.Context,
+// New returns a new runnable frontend
+func New(ctx context.Context,
 	baseLog *logrus.Entry,
 	_env env.Interface,
 	db *database.Database,
-	apis map[string]*api.Version,
+	apis map[string]*api.Version, // TODO: Give "apis" a default and override it in tests by modifying struct
```

Comment:
@asalkeld I included one of your suggestions here as a TODO. Do you think it's in the scope of this PR to fix here?

Reply:
do one thing at a time